projects/adrv902x/scripts/json2cstring

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# Usage example:
+#     json2cstring path/to/file.json
+#
+CFILE="${1%.json}.h"
```

Comment:
this file should go under tools/scripts as it does something that can be used in many projects

Reply:
Done.